python/doc/examples/data_analysis/smoothing_mixture.ipynb

```diff
@@ -0,0 +1,524 @@
+      ]
+     },
+     "execution_count": 12,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "graph = distribution.drawPDF()\n",
+    "curve = fit.drawPDF()\n",
+    "graph.add(curve)\n",
+    "graph.setColors([\"dodgerblue3\", \"darkorange1\"])\n",
+    "graph.setLegends([\"Mixture\", \"Kernel smoothing\"])\n",
+    "graph.setLegendPosition(\"topleft\")\n",
+    "graph"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We see that the result is similar to the previous one. However, it seems that the mixture is not so well approximated."
```

Comment:
```suggestion
    "We see that the result of the kernel smoothing approximation of the mixture is similar to the previous one and could be improved."
```
There is some ambiguity in the original formulation because it is not clear we are comparing the current kernel smoothing to the previous one.